packages/compute-baseline/src/browser-compat-data/supportStatements.ts

```diff
@@ -33,23 +33,35 @@ export class SupportStatement {
     this.feature = feature;
   }
 
+  /**
+   * An array of `FlagStatement` objects. If there are no flags, then the array
+   * is empty.
+   */
   get flags(): FlagStatement[] {
     return this.data?.flags ?? [];
   }
 
+  /**
+   * A `true` value, if support exists but may be incompatible with other
+   * implementations; otherwise, `false`.
+   */
   get partial_implementation(): boolean {
-    // Strictness guarantee: unset partial_implementation returns false
     return this.data?.partial_implementation ?? false;
   }
 
   /**
-   * Get the `version_added` value
+   * A version string for the first supporting release. If the feature was never
+   * supported, then it is `false`.
    */
-  get version_added() {
+  get version_added(): string | false {
     return this.data?.version_added;
```

Comment:
To ensure that null or undefined aren't returned:

```suggestion
    return this.data?.version_added ?? false;
```